### DISCUSSION TITLE
Notifications on shared albums/conversation

### DISCUSSION THREAD

**Discussion by shqipesson** [immich-app/immich#13878] [Feature Request]
### I have searched the existing feature requests to make sure this is not a duplicate request.

- [X] Yes

### The feature

The feature with shared albums and conversations is one of the best feature in my opinion because it is one thing keeping me on Google photos.. and immich has it but when adding or writing something there the user don't seem to get a notification so it defeats the function

### Platform

- [ ] Server
- [ ] Web
- [X] Mobile

**Comment by chamdim** [comment]
Have you set up email notifications using this [guide](https://immich.app/docs/administration/email-notification/)?

**Reply by eerison** [reply]
The user won't be notified when someone write a comment in the photo.

And there isn't an option to tag the user.

**Comment by Eric1701A** [comment]
With the Message feature available for each photo, add a way for other users to see easily that a photo has a new comment on it. Preferably in three different methods... 1) traditional red notification dot on the photo thumbnail itself. 2) traditional red notification dot on the album that contains the commented photo. 3) Message log for album owner or admin that contains a linked photo name and the comment.

**Comment by dkerlee** [comment]
this shouldn't be limited to Comments/hearts/likes, it should include _newly uploaded assets_ to the share album.

**Reply by wBrhy2** [reply]
Yup. It's why I'm looking at finding an alternative. I regret immich.

**Reply by NickOveracker** [reply]
> Yup. It's why I'm looking at finding an alternative. I regret immich.

This is a project under rapid development. It's not like we aren't getting important new features every few weeks.

Notifications for comments and other events are almost certainly in its future.

**Comment by dkerlee** [comment]
I've shopped around at image hosting stuff, and nothing comes close to this project with the apps, backend, and now community. I continue to see Alex Tran himself occasionally comment on our reddit posts. All indications point to this being a great project, and evolving into an A-list image hosting service.

Where we go from here is identifying things that are mistakes/bugs, and new features we'd like to see. We, the community take part in the development. Frequently we just can't help ourselves on slapping a personal importance to the request. "I can't believe this hasn't been implemented yet," "this feature sucks compared to service XXX," and other value statements.

There is a system for Feature Requests and feedback. Participating in those threads by just slapping a "+1" on them, or giving them a thumbs up, shows that individuals agree and want the feature. That's the system in place for identifying community requests. The devs will likely take that into consideration on where to put their efforts.

https://immich.app/roadmap This is where you want to see your request. I'm most excited for "grandma mode" aka Smart Albums. I can make an album with my daughter's face, and direct my grandparents to that stream of pics.

Below that is the github Feature Requests. There are also discussions in that forum.
https://github.com/immich-app/immich/discussions/categories/feature-request

and below that, I'd say is our reddit thread. A bit more approachable than the github location.
https://www.reddit.com/r/immich/

Immich, and FUTO, are re-inventing their licensing scheme and how to actually employ people to work on this project. A self-sustaining software package. We want a good project that takes time, and code, and iteration for user experience. I don't expect Immich to be on par with Google or Apple Photos. But in the privacy first self-hosted space?: They are knocking it out of the park. I'm a huge fan, I'll continue to host my own instance, pay them in a way they request, and participate in Immich development as a community member. I am 100% certain this project will be one of the best hosting packages out there, and we're on that path today.

**Reply by wBrhy2** [reply]
Good response. I feel like a jerk for my previous answer. 

Grandma mode sounds awesome.
